### PR TITLE
Use f-strings on constants for better readability

### PR DESCRIPTION
**Pull request by blaisewf**
It's not a very significant change, but I think it makes it easier to read/modify the url more easily

### CODE REVIEW

**Review by andresayac — APPROVED** [review]

**Review by alxpez — APPROVED** [review]

**Comment by rany2** [comment]
Please fix CI by create a base URL constant and using that for both voice list and websocket URL. This way you won't hit the maximum characters/line limit.

File: src/edge_tts/constants.py [hunk 1]
```diff
@@ -3,13 +3,7 @@
...
-    "wss://speech.platform.bing.com/consumer/speech/synthesize/"
-    + "readaloud/edge/v1?TrustedClientToken="
-    + TRUSTED_CLIENT_TOKEN
-)
-VOICE_LIST = (
-    "https://speech.platform.bing.com/consumer/speech/synthesize/"
-    + "readaloud/voices/list?trustedclienttoken="
-    + TRUSTED_CLIENT_TOKEN
-)
+BASE_URL = "https://speech.platform.bing.com/consumer/speech/synthesize"
```
Comment (rany2) [inline src/edge_tts/constants.py:6]:
```suggestion
BASE_URL = "speech.platform.bing.com/consumer/speech/synthesize"
```

File: src/edge_tts/constants.py [hunk 2]
```diff
@@ -3,13 +3,7 @@
...
+
+WSS_URL = f"wss://{BASE_URL}/readaloud/edge/v1?TrustedClientToken={TRUSTED_CLIENT_TOKEN}"
+VOICE_LIST = f"{BASE_URL}/readaloud/voices/list?trustedclienttoken={TRUSTED_CLIENT_TOKEN}"
```
Comment (rany2) [inline src/edge_tts/constants.py:9]:
```suggestion
VOICE_LIST = f"https://{BASE_URL}/readaloud/voices/list?trustedclienttoken={TRUSTED_CLIENT_TOKEN}"
```

**Review by rany2 — CHANGES_REQUESTED** [review]
Required because WSS uses the wss:// scheme instead.

**Review by rany2 — APPROVED** [review]

**Comment by rany2** [comment]
Thanks!

**Pull request merged**